NEWS.md

```diff
@@ -1,6 +1,10 @@
 shiny 1.3.2.9000
 =======
 
+### Improvements
+
+* An informative warning is now thrown for mis-specified (date) strings in `dateInput()`, `updateDateInput()`, `dateRangeInput()`, and `updateDateRangeInput()`. ([#2402](https://github.com/rstudio/shiny/pull/2402))
```

Comment:
Should start with `Fixed #2402` or "Closed" or "Resolved". Also, this PR is 2403.